employee-onboarding-A2A-reference-implementation/src/mcp/it_mcp_server.py

```
@@ -0,0 +1,734 @@
+        "subject_token": subject_token,
+        "subject_token_type": "urn:ietf:params:oauth:token-type:access_token",
+        "actor_token": subject_token,  # Same token acts as actor for scope narrowing
+        "actor_token_type": "urn:ietf:params:oauth:token-type:access_token",
+        "scope": target_scope,
+    }
+    
+    # Add audience if specified for API-specific token binding
+    if target_audience:
+        data["audience"] = target_audience
+
+    async with httpx.AsyncClient(verify=False, timeout=10.0) as client:
```

Ruff (0.15.2)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)